jdaviz/configs/imviz/tests/test_delete_data.py

```diff
@@ -66,17 +65,16 @@ def test_delete_with_subset_wcs(self):
         assert len(self.imviz.app.data_collection.external_links) == 2
 
         # Check that the reparenting and coordinate recalculations happened
-        assert subset1.subset_state.xatt.parent.label == "has_wcs_2[SCI,1]"
-        assert_allclose(subset1.subset_state.center(), (3, 2))
+        assert subset1.subset_state.xatt.parent.label == "Default orientation"
+        assert_allclose(subset1.subset_state.center(), (1.75, 1.75))
 
-        assert subset2.subset_state.xatt.parent.label == "has_wcs_2[SCI,1]"
-        assert_allclose(subset2.subset_state.roi.xmin, 1)
-        assert_allclose(subset2.subset_state.roi.ymin, 0, atol=1e-6)
-        assert_allclose(subset2.subset_state.roi.xmax, 3)
-        assert_allclose(subset2.subset_state.roi.ymax, 2)
+        assert subset2.subset_state.xatt.parent.label == "Default orientation"
+        assert_allclose(subset2.subset_state.roi.xmin, -0.25)
+        assert_allclose(subset2.subset_state.roi.ymin, -0.25, atol=1e-6)
```

Comment:
Do you still need this?
```suggestion
        assert_allclose(subset2.subset_state.roi.ymin, -0.25)
```

Reply:
Probably not!